build/commands/lib/build_fuzzer.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+const build = require('./build')
+
+const buildFuzzer = (fuzzer_test_target, options) => {
+  options.use_libfuzzer = true
+  options.is_asan = true
+  options.target = fuzzer_test_target
+  options.is_component_build = false
+
+  build('Fuzzer', options)
```

Comment:
I think it would be better to combine run and build into a single fuzzer.js file. I think it's easier to follow that way and also we don't duplicate 'Fuzzer' https://github.com/brave/brave-core/pull/16479/files#diff-34c6b846e7c8256a60d04b9d484168e32ef8c0a7aa5216a875ef99518c772661R17

Reply:
Makes sense, Done.